airflow/hooks/dbapi.py

```diff
@@ -324,6 +324,7 @@ def insert_rows(self, table, rows, target_fields=None, commit_every=1000, replac
                         lst.append(self._serialize_cell(cell, conn))
                     values = tuple(lst)
                     sql = self._generate_insert_sql(table, values, target_fields, replace, **kwargs)
+                    self.log.info(f"Generated sql: {sql}")
```

Comment:
This might be a bit too agressive. Please change it to debug, also use "%" format of log.* rather than f-string for performance.

Reply:
Fixed. Thanks, @potiuk